File: app/src/test/java/com/aoeiuv020/kotlindemo/AndroidLibraryClassTest.java
package com.aoeiuv020.kotlindemo;

import com.aoeiuv020.androidlibrary.AndroidLibraryClass;

import org.junit.Test;
import org.slf4j.simple.SimpleLogger;

public class AndroidLibraryClassTest {
    static {
        System.setProperty(SimpleLogger.LOG_KEY_PREFIX + "JavaLibrary", "trace");
        System.setProperty(SimpleLogger.LOG_KEY_PREFIX + "AndroidLibrary", "trace");
    }

    @Test
    public void init() {
        var androidLibraryClass = new AndroidLibraryClass();
        androidLibraryClass.init();
        System.out.println(AndroidLibraryClass.androidLibraryBean);
        System.out.println(AndroidLibraryClass.javaLibraryBean);
    }
}

File: app/src/test/java/com/aoeiuv020/kotlindemo/AndroidLibraryClassTest.kt
package com.aoeiuv020.kotlindemo

import com.aoeiuv020.androidlibrary.AndroidLibraryClass
import org.junit.Test
import org.slf4j.simple.SimpleLogger

class AndroidLibraryClassTest {
    init {
        System.setProperty(SimpleLogger.LOG_KEY_PREFIX + "JavaLibrary", "trace")
        System.setProperty(SimpleLogger.LOG_KEY_PREFIX + "AndroidLibrary", "trace")
    }

    @Test
    fun init() {
        val androidLibraryClass = AndroidLibraryClass()
        androidLibraryClass.init()
        println(AndroidLibraryClass.androidLibraryBean)
        println(AndroidLibraryClass.javaLibraryBean)
    }
}